server/scsynth/scsynth_main.cpp

```diff
@@ -44,6 +44,38 @@ inline int setlinebuf(FILE* stream) { return setvbuf(stream, (char*)0, _IONBF, 0
 
 #endif
 
+#ifdef _WIN32
+// Add a warning when Windows Defender delays scsynth boot by 60+ seconds
+// cf. github issue #4368
+#    include <iostream>
+#    include <thread>
+#    include <mutex>
+#    include <chrono>
+bool bScsynthHasBooted = false;
+std::mutex bScsynthHasBootedMutex;
+std::condition_variable bScsynthHasBootedConditionVariable;
+
+void scsynthHasBooted() {
+    {
+        const std::lock_guard<std::mutex> lock(bScsynthHasBootedMutex);
+        bScsynthHasBooted = true;
+    }
+    bScsynthHasBootedConditionVariable.notify_all();
+}
+
+void displayWarningIfScsynthHasNotBooted() {
```

Comment:
these functions should both be marked `static`

Reply:
Interesting ! I did not know the difference and I learnt that [static function names are not visible to other files ](https://stackoverflow.com/questions/7813593/what-is-the-advantage-of-static-function). Thanks !

Reply:
yep! it's a small cleanliness thing, and also helps to communicate that these functions and variables won't be used in any other file.